hooks/useSelectedAccount.ts

```diff
@@ -0,0 +1,18 @@
+import { useMemo } from "react";
+import { cvmToAddress } from "@cennznet/types/utils";
+import { useCENNZWallet } from "@/providers/CENNZWalletProvider";
+import { useMetaMaskWallet } from "@/providers/MetaMaskWalletProvider";
+import { useWalletProvider } from "@/providers/WalletProvider";
+import { InjectedAccountWithMeta } from "@polkadot/extension-inject/types";
+
+export default function useSelectedAccount(): Partial<InjectedAccountWithMeta> {
```

Comment:
fancy hook use! 👏 

it should be `Pick<InjectedAccountWithMeta, "address">` as return type, given that's all we return back